src/Core/Checkout/Cart/Facade/CartFacade.php

```diff
@@ -22,7 +22,7 @@
  * @script-service cart_manipulation
  */
 #[Package('checkout')]
-class CartFacade
+class CartFacade implements \Countable
```

Comment:
why does phpstan care about that?

Reply:
in the tests the facade is directly used on the assertCount method. it worked, because it implements a `count` method. but right after that assert, PHPStan is no longer able to determine the type of the facade, as it seems, that only class that are implementing the Countable interface should be valid inputs.

Reply:
ah makes sense